spec/models/chargeback_vm_spec.rb

```diff
@@ -190,6 +190,22 @@
 
         subject { ChargebackVm.build_results_for_report_ChargebackVm(options).first.first }
 
+        context 'when Vm(resource) is destroyed' do
```

Comment:
Vm(resource) is hard to follow if you're not familiar with chargeback.  Would `when the Vm resource of a consumption is destroyed` be better?

Reply:
yes, it sounds better to me 👍